app/views/admin/tags/_tag.html.haml

```diff
@@ -0,0 +1,28 @@
+.batch-table__row{ class: [!tag.requires_review? && !tag.usable? && 'batch-table__row--muted'] }
+  .batch-table__row__content.batch-table__row__content--padded.pending-account
+    .pending-account__header
+      %strong
+        = link_to admin_tag_path(tag.id) do
+          = t('admin.tags.tag_name', name: tag.display_name)
```

Comment:
Not sure using a translation here is correct, but it felt more correct than `= '#' + tag.display_name` or something like that.

Reply:
Interesting we have similar logic here: https://github.com/mastodon/mastodon/blob/74ead7d10698c7f18ca22c07d2f04ff81f419097/app/serializers/activitypub/hashtag_serializer.rb#L15

Perhaps it makes sense to have a `Tag#formatted_name` method?

Reply:
I think it would!

And I would like that a lot better than the translation based approach.

Reply:
Okay, have implemented this, and also updated everywhere that we had this "##{...}" syntax for tags.